<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pluginSettingDialog</class>
 <widget class="QDialog" name="pluginSettingDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统监控插件设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="displayContentSettingGroupBox">
     <property name="title">
      <string>高效模式显示内容设置</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>41</y>
        <width>147</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="onlyNetSpeedRadioButton">
         <property name="text">
          <string>只显示网速</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="onlyCPUMEMRadioButton">
         <property name="text">
          <string>只显示CPU、内存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="showAllRadioButton">
         <property name="text">
          <string>显示CPU、内存和网速</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fashionDisplayContentSettingGroupBox">
     <property name="title">
      <string>时尚模式显示内容设置</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>57</y>
        <width>123</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="fashionOnlyCPUMEMRadioButton">
         <property name="text">
          <string>只显示CPU、内存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fashionOnlyNetSpeedRadioButton">
         <property name="text">
          <string>只显示网速</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pluginSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pluginSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
